bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -191,9 +191,7 @@
 		</supported-bridge-type-refs>
 		<label>T13</label>
 		<description>Digital Input</description>
-
 		<channels>
-			<channel id="stateOnOff" typeId="stateOnOff"/>
```

Comment:
Since this was the only usage of channel type id `stateOnOff`, the channel type can be removed as well.